core/templates/pages/admin-page/roles-tab/admin-roles-tab.directive.ts

```diff
@@ -161,17 +161,19 @@ angular.module('oppia').directive('adminRolesTab', [
           }
           ctrl.setStatusMessage('Processing query...');
           AdminTaskManagerService.startTask();
+          ctrl.contributionReviewersResult = {};
           if (formResponse.filterCriterion === USER_FILTER_CRITERION_ROLE) {
             AdminBackendApiService.viewContributionReviewersAsync(
               formResponse.category, formResponse.languageCode
             ).then((usersObject) => {
-              ctrl.result.usernames = usersObject.usernames;
+              ctrl.contributionReviewersResult.usernames =
+                usersObject.usernames;
```

Comment:
```suggestion
              ctrl.contributionReviewersResult.usernames = (
                usersObject.usernames);
```

Break lines after parenthesis

Reply:
Done.